### PR TITLE
[EDX-413] - basics - what

### PR DESCRIPTION
**Pull request by Sapples**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

Add a page giving an overview of the basics of Ably.

* [Jira ticket](https://ably.atlassian.net/browse/EDX-413).

Note - new branch and PR after rebase of IA update for clarity, [original PR](https://github.com/ably/docs/pull/1746)

## Review

Check /basics/ably in the new toolchain.

### CODE REVIEW

File: content/basics/ably.textile [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: What is Ably?
+meta_description: "Ably is a realtime platform delivering powerful digital realtime experiences through a publish/subscribe network."
+---
+
+Ably’s realtime platform provides a scalable, fault-tolerant and worldwide publish/subscribe(pub/sub) network. Providing SDKs in a variety of programming languages with both REST and realtime interfaces Ably helps you get connected.
```
Comment (tbuntel) [inline content/basics/ably.textile:6]: This should align with our core messaging. 
"Ably is the platform to power synchronized digital experiences in realtime. Ably's SDKs and APIs power realtime capabilities like Chat, Data Broadcast, Data Synchronization, Multiplayer Collaboration, and Notifications."

File: content/basics/ably.textile [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+This section will provide a short introduction to some of the key features and APIs of Ably, giving you an idea of what you can expect to build with Ably and how Ably's infrastructure helps you succeed.
+
+h2(#what). What can you build with Ably?
+
+Ably enables the distribution of realtime data to multiple users using the pub/sub messaging pattern with WebSockets, enabling you to build realtime experiences for your applications without having to build your own realtime infrastructure. Common use cares are:
+
+* Live Chat - chat for customer support or community engagement
```
Comment (tbuntel) [inline content/basics/ably.textile:14]: "chat for customer support, live events, or community engagement"

File: content/basics/ably.textile [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Ably provides a number of features that enhance realtime experiences, some of the key Ably features are: 
+
+* "Pub/sub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). Pub/sub with channels
+
+Ably uses the pub/sub architectural pattern to enable any number of publishers to publish data and any number of subscribers to subscribe to that data. This works with Ably's "concept of channels":/channels, enabling publishers to publish to a single location and subscribers to subscribe to a single source.
```
Comment (tbuntel) [inline content/basics/ably.textile:31]: Would it be good to give an example here? something like "pub/sub could be used to power live data feeds such as sports scores, travel updates, stocks, currency or auction prices."

File: content/basics/ably.textile [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+h3(#presence-and-occupancy). Presence and occupancy
+
+Ably’s "presence":/presence-occupancy/presence feature enables clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events, such as a client entering or leaving the presence set. Each member in the presence set can have data associated with them. You can also request a list of clients or devices that are connected to a channel at a particular point in time via the REST API.
```
Comment (tbuntel) [inline content/basics/ably.textile:35]: Again, an example here could be "Presence could be used to power a live avatar stack to show what users are present in a virtual space (e.g. whiteboard, document, or worksheet)"

File: content/basics/ably.textile [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+h3(#message-storage-and-history). Message storage and history
+
+By default messages are only stored for two minutes on Ably’s servers. Messages are retrieved automatically by the SDK in the event of a lost connection within these two minutes. However, if you've enabled message persistence and the disconnection is longer than two minutes you can use the history feature to retrieve messages.
+
+Ably's "message storage and history":/storage-history enables messages to be accessed after initially being sent. When enabled you can retrieve the last message sent to a channel using rewind, or enable persistence of all messages for 24 hours for the free account tier or 72 hours for paid account tiers. These messages can be retrieved using history. You can also persist messages to external storage services such as AWS through the use of "Ably Integrations":/integrations.
+
+h3(#push-notifications). Push notifications
+
+Ably's "Push Notifications":/push feature enables messages to be pushed to subscribers without needing a device to maintain a connection to Ably.
```
Comment (tbuntel) [inline content/basics/ably.textile:47]: "enables messages to be pushed to subscribers on mobile devices without needing to maintain a connection to Ably.

File: content/basics/ably.textile [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+h3(#interfaces). SDK interfaces
+
+Ably provides "both realtime and REST interfaces":/basics/use-ably. 
+
+The "realtime interface":/basics/use-ably#realtime-interface establishes and maintains a constant connection to Ably, enabling realtime messages to be sent both to and from Ably. Using multiplexed "WebSocket":/https://ably.com/topic/websockets connections to enable multiple channels to share a single connection without having to fall back to inefficient techniques like HTTP polling.
```
Comment (tbuntel) [inline content/basics/ably.textile:53]: Incomplete sentence. S/b "Using multiplexed "WebSocket":/https://ably.com/topic/websockets connections enables multiple channels to share a single connection without having to fall back to inefficient techniques like HTTP polling.

File: content/basics/ably.textile [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+The "REST interface":/basics/use-ably#rest-interface uses HTTP protocols and is effectively stateless, removing the need to retain session information or any in-flight request states. Although the REST interface cannot subscribe to data published in real time, it can be used to publish and retrieve realtime data.
+
+There are a number of "Ably SDKs":LINK-ME in different programming languages to match your needs.
+
+h2(#infrastructure). Ably's realtime infrastructure
+
+Ably's infrastructure focuses on the "Four Pillars of Dependability":#infrastructure: performance, integrity, reliability, and availability.
+
+Ably has a Global Network of 7 geographically-distributed core routing datacenters and 307 edge acceleration points-of-presence(PoPs). Ably's Global Network ensures unrivalled realtime messaging, data streaming performance, availability, and reliability.
```
Comment (tbuntel) [inline content/basics/ably.textile:63]: 385 edge acceleration points-of-presence (per https://ably.com/network)

**Review by tbuntel — CHANGES_REQUESTED** [review]
@m-hulbert I have a large number of comments. Might it be better to work on this in a google doc?

**Comment by m-hulbert** [comment]
Old PR - closing in favour of a new one.

**Pull request closed without merging**